src/ontology/mondo-edit.obo

```diff
@@ -417738,8 +417739,8 @@ synonym: "GSD type 2, late onset" EXACT [Orphanet:420429]
 synonym: "GSD type 2, late-onset" EXACT [Orphanet:420429]
 synonym: "GSD type II, late onset" EXACT [Orphanet:420429]
 synonym: "GSD type II, late-onset" EXACT [Orphanet:420429]
+synonym: "LOPD" EXACT ABBREVIATION [Orphanet:420429]
 synonym: "Pompe disease, late onset" EXACT [Orphanet:420429]
-synonym: "Pompe disease, late-onset" EXACT [Orphanet:420429]
```

Comment:
I removed the duplicate synonym

Reply:
This synonym is not duplicated. There is a dash between "late" and "onset", which is not in the other synonym. I know it looks like a detail, but if people use a dash in their search, they might not find this term. 
Please reinstate this synonym.